src/commands/version.c

```diff
@@ -23,7 +23,7 @@
 static void
 show_version (void)
 {
-	g_print ("%s version: %s\n", PACKAGE_NAME, PACKAGE_VERSION);
+	g_print ("%s version: %s-networking-poc\n", PACKAGE_NAME, PACKAGE_VERSION);
```

Comment:
If that is indeed the only non-rebase change (please could you specify in subsequent PRs to make life easier)...

lgtm.